Add support template (#4)
CODAIT/max-central-repo: README.md

4. [Serverless App](https://developer.ibm.com/tutorials/deploy-a-model-asset-exchange-microservice-on-red-hat-openshift/)
5. [CodePen](https://codepen.io/collection/DzdpJM/#)

# Contact Us

If you have any questions that you would like to discuss with us:

1. For general discussion, you are welcome to open an issue in [our issue tracker](https://github.com/CODAIT/max-central-repo/issues/new).

> 2. For discussion with a particular model, you are welcome to open an issue in the corresponding repository. You can find a list of the repositories at https://github.com/CODAIT/max-central-repo/blob/master/max-model-status.md

Reviewer: ```suggestion
2. For a question about a particular model, you are welcome to open an issue in the corresponding repository. You can find a list of the repositories [here](https://github.com/CODAIT/max-central-repo/blob/master/max-model-status.md).
```

1. To start a general discussion create an issue in this repo.
2. For model-specific questions, go to the corresponding model repository and create an issue.
3. For those who want to contribute a model to MAX, here is a quick summary of the process:

>    - To contribute code, documentation, or tests, please submit a pull request in [the model's repository](https://github.com/CODAIT/max-central-repo/blob/master/max-model-status.md) using the [template](PULL_REQUEST_TEMPLATE.md) to this GitHub repository. 

Reviewer 2: This doesn't seem quite the optimal way to do things - are you saying the flow is: look at the PR template file and use that to open a PR against a specific model's repo?

To me it would then make more sense to have a PR template in each model repo.

Author: Good point. See #7
